apps/nextjs/src/pages/app/index.tsx

```diff
@@ -1,39 +1,26 @@
 import type { NextPage } from "next";
 import { useUser } from "@clerk/nextjs";
-import { useEffect, useState } from "react";
 import { trpc } from "../../utils/trpc";
-import ProfileSetup from "../../components/ProfileSetup";
-import { Profiles } from "@pickle-app/db";
 import { AppHeaderMenu } from "../../components/HeaderMenu";
+import { useRouter } from "next/router";
 
 const App: NextPage = () => {
-  const [userProfile, setUserProfile] = useState<Profiles>();
-  const { user } = useUser();
-  console.log("user", user);
+  const router = useRouter();
+  const { user, isSignedIn } = useUser();
+  const userId = user?.id ?? "";
 
-  // check if profile exists
-  const { data: profile } = trpc.profile.getById.useQuery(user?.id);
-  console.log("profile", profile);
+  const { isSuccess, data: profile } = trpc.profile.getById.useQuery(userId, {
+    enabled: !!isSignedIn,
```

Comment:
Do we want to pick a pattern and stick with it? I don’t think we use the double bang in a lot of places. What are your thoughts?

Reply:
Yep we don't need the double bang since `isSignedIn` is already a boolean.